tests/mockers.py

```diff
@@ -55,10 +55,13 @@ def execute(self, command: str) -> "GitRepositorySandbox":
         subprocess.check_output(command, stderr=subprocess.STDOUT, shell=True)
         return self
 
-    def new_repo(self, *args: str) -> "GitRepositorySandbox":
+    def new_repo(self, *args: str, switch_dir_to_new_repo: bool = True) -> "GitRepositorySandbox":
```

Comment:
How many times is this method called across the project? TBH I'd prefer if `switch_dir_to_new_repo` param did NOT have a default value... let's make the behavior as explicit as possible

Reply:
Huh okay... let's live with that then